scripts/irods/lib.py

```diff
@@ -717,3 +717,36 @@ def log_command_result(command, stdout, stderr, ec):
     print('  stderr:')
     output = [indent_prefix + line for line in stderr.splitlines()]
     print(os.linesep.join(output) + os.linesep)
+
+def remove_file_if_exists(filename):
+    if os.path.exists(filename):
+        os.remove(filename)
+
+def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
+    # do not care about true randomness
+    # note that this method does not use up system entropy
+    random.seed(5)
+    bytes_written = 0
+    buffer = buffer_size * [0x78]       # 'x' - bytearray() below appears to require int instead
+                                        #       of char which was valid in python2
+    with open(f_name, "wb") as out:
```

Comment:
Does it help/work to open the file in text mode (`w` or `wt`) rather than binary mode (`wb`)? Might be able to just write characters instead of doing the `bytearray` thing that way. Then again, it might not. Just thought I'd mention

Reply:
I am not sure.  I fooled around with it for a while when I first wrote it to get it to work.

Reply:
Okay, if this works, then I think the interface part looks fine for `lib.py`. Feel free to resolve this